app/scripts/constants/sentry-state.ts

```diff
@@ -440,6 +440,7 @@ export const SENTRY_UI_STATE = {
     ///: END:ONLY_INCLUDE_IF
     switchedNetworkDetails: false,
     switchedNetworkNeverShowMessage: false,
+    isRampCardClosed: false,
```

Comment:
This is set in the wrong place, it should be in the `AppStateController` property of `SENTRY_BACKGROUND_STATE` above.

This section here is for UI-specific properties in the `metamask` slice (which this is not, since it's in a controller)